.eslintignore

```diff
@@ -2,6 +2,33 @@
 node_modules
 packages/core/src/lib/markdown-it/*
 packages/core/src/lib/markdown-it-shared/*
-packages/vue-components
 
 !.eslintrc.js
+
+# vue-components
```

Comment:
Just wondering - is there any reason why we don't just use `packages/vue-components` here?

Reply:
👍 fixed

Reply:
> Some questions/thoughts:
> - Do you know why the FontAwesome version was bumped? Ideally, we should retain the original version in this PR to keep these concerns separate and avoid unnecessary diffs.

we previously installed the core package transitively via the cli package, so there was no `package-lock` there. so the latest semantic ver gets installed.

I'll get out a separate pr to update it though, then rebase this one on top of that

> - I feel that we should keep the `README.md` in the root. This can help new devs who first visit our repo some information on the project and how to get started at first glance. Maybe we should set up a new README for `cli` and each of the other leaf packages, with a short explanation on what each package does.

🤦‍♂️ unintentional. Will revert and add some brief readmes for `cli/core` after the `npmignore` commit during the rebase above